<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerWindow</class>
 <widget class="QMainWindow" name="ImageViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>115</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>115</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>115</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../example.qrc"/>
 </resources>
 <connections/>
</ui>
